demisto_sdk/commands/test_content/test_use_case/test_use_case.py

```diff
@@ -0,0 +1,661 @@
+    ):
+        """Uploads a JSON object to a specified path in the GCP bucket.
+
+        Args:
+          original_file_path: The path to the JSON file to upload.
+          repository_name: The name of the repository within the bucket.
+          file_name: The desired filename for the uploaded JSON data.
+          logging_module: Logging module to use for upload_result_json_to_bucket.
+        """
+        logging_module.info("Start uploading test use case results file to bucket")
+
+        storage_client = storage.Client.from_service_account_json(self.service_account)
+        storage_bucket = storage_client.bucket(self.artifacts_bucket)
+
+        blob = storage_bucket.blob(
+            f"content-test-use-case/{repository_name}/{file_name}"
+        )
+        blob.upload_from_filename(
+            original_file_path.as_posix(),
+            content_type="application/xml",
+        )
+
+        logging_module.info("Finished uploading test use case results file to bucket")
+
+
+class BuildContext:
+    def __init__(
+            self,
+            nightly: bool,
+            build_number: Optional[str],
+            logging_module: ParallelLoggingManager,
+            cloud_servers_path: str,
+            cloud_servers_api_keys: str,
+            service_account: Optional[str],
+            artifacts_bucket: Optional[str],
+            cloud_url: Optional[str],
+            api_key: Optional[str],
+            auth_id: Optional[str],
+            inputs: Optional[List[Path]],
+            machine_assignment: str,
+            lcas_id: str,
+            ctx: typer.Context,
+    ):
+        self.logging_module: ParallelLoggingManager = logging_module
+        self.ctx = ctx
+
+        # --------------------------- overall build configuration -------------------------------
+        self.is_nightly = nightly
+        self.build_number = build_number
+        self.lcas_id = lcas_id
+
+        # -------------------------- Manual run on a single instance --------------------------
+        self.cloud_url = cloud_url
+        self.api_key = api_key
+        self.auth_id = auth_id
+        self.inputs = inputs
+
+        # --------------------------- Machine preparation -------------------------------
+
+        self.cloud_servers_path_json = get_json_file(cloud_servers_path)
+        self.cloud_servers_api_keys_json = get_json_file(cloud_servers_api_keys)
+        self.machine_assignment_json = get_json_file(machine_assignment)
+
+        # --------------------------- Testing preparation -------------------------------
+
+        self.tests_data_keeper = TestResults(
+            service_account,
+            artifacts_bucket,
+        )
+
+        # --------------------------- Machine preparation logic -------------------------------
+
+        self.servers = self.create_servers()
+
+        # --------------------------- Env Setup -------------------------------
+
+        logger.info("installing pyxdr...")
```

Comment:
```suggestion
        logger.debug("installing pyxdr...")
```
generally go over everything and ensure logs are info only for "user messages" and not for "developer messages"
I recommend reading the job logs in the gitlab UI to easily understand which info logs we actually want to be debug